dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_test.cu

```diff
@@ -50,11 +52,11 @@ cv::Mat bgr2rgb(const cv::Mat& img) {
 template <typename GTestParams>
 class JpegDistortionTestGPU : public ::testing::Test {
   using T = typename GTestParams::T;
-  static constexpr bool vert_subsample = GTestParams::vert_subsample;
-  static constexpr bool horz_subsample = GTestParams::horz_subsample;
-  static constexpr bool use_real_images = true;
-  static constexpr bool perf_run = false;
-  static constexpr bool dump_images = false;
+  const bool vert_subsample = GTestParams::vert_subsample;
+  const bool horz_subsample = GTestParams::horz_subsample;
```

Comment:
Now that subsampling is a run-time parameters, there's no need to make them part of the type.

Reply:
done